README.md

```diff
@@ -1,9 +1,27 @@
-# condor
+# Condor Central Manager and Schedd
 
-KBase specific Condor scheduler image.
+KBase specific condor scheduler image.
 Based on the htcondor/cm:9.X images
 
-## Supported ENV Variables
+## New behavior
```

Comment:
Ugh, looks like I forgot to update this with the variables from pre-exec.sh. Will have to do that next